rules_gapic/README.md

```diff
@@ -71,6 +71,17 @@ The rules will call `gapic-generator` and do all the necessary pre- and post- ge
 
 5. **`ruby_gapic_assembly_pkg`** - Ruby **macro** which accepts the previously built `ruby_proto_library`, `ruby_grpc_library` and `ruby_gapic_library` artifacts as arguments and packages them into an idiomatic (for Ruby) package which is ready for opensourcing and is independent from Bazel.
 
+#### C#
+1. **`csharp_proto_library`** - C# **macro**, which generates C# protobuf stubs by calling protobuf compiller with `--csharp_out` parameter.
```

Comment:
nit: typo"compiller" -> "compiler"
(and below)

Reply:
Fixed (thanks, I was propagating this typo to all other languages =)